design/parallel-commit/initial-design.md

```diff
@@ -51,35 +68,105 @@ We also need to be able to clean up dead parallel commit transactions:
   }
 ```
 
-## First iteration
-
 ## Evolution to long term solution
 
+The framework of parallel commit will be in place at the end of the first iteration. In a later iteration we should improve the temporary locking mechanism, See [parallel-commit-solution-ideas.md](parallel-commit-known-issues-and-solutions.md) for possible improvements.
+
+### Open questions
+
+* There is a problem if the commit_ts calculated during a resolve lock is > pd’s latest ts + 1, that is separate from the problem of non-unique timestamps (but has the same root cause). ([#21](https://github.com/tikv/sig-transaction/issues/21))
+* Replica read. ([#20](https://github.com/tikv/sig-transaction/issues/20))
+* Interaction with CDC. ([#19](https://github.com/tikv/sig-transaction/issues/19))
+* Interaction with binlog. ([#19](https://github.com/tikv/sig-transaction/issues/19))
+* Missing schema check.
+* Interaction with large transactions.
```

Comment:
Is there any issue about large transaction?

Reply:
No idea. I think it was mentioned at some point because of the extra timestamp. Not sure if there is actually any issue

Reply:
No, as long as other transactions don't push parallel commit transactions' commit_ts.